build/CopyFilesToStagingDir.ps1

```diff
@@ -55,6 +55,27 @@ PublishFile $FullBuildOutput\RestartAgent\RestartAgent.exe $FullPublishDir\Micro
 PublishFile $FullBuildOutput\RestartAgent\RestartAgent.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
 PublishFile $FullBuildOutput\DeploymentAgent\DeploymentAgent.exe $FullPublishDir\Microsoft.WindowsAppRuntime\
 PublishFile $FullBuildOutput\DeploymentAgent\DeploymentAgent.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+#
+## Publish Kozani Artifacts
+### Kozani WINMDs in MSIX Framework package
+PublishFile $FullBuildOutput\KozaniAppGraph\StrippedWinMD\Microsoft.Kozani.AppGraph.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\StrippedWinMD\Microsoft.Kozani.Package.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\StrippedWinMD\Microsoft.Kozani.SendToLocal.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\StrippedWinMD\Microsoft.Kozani.SendToRemote.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\StrippedWinMD\Microsoft.Kozani.Settings.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniManagerRuntime\StrippedWinMD\Microsoft.Kozani.ManagerRuntime.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+#
+### Kozani DLLs/EXEs in MSIX Framework package
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
```

Comment:
Why do the PDBs get published into the fwk package? Is that required for debugging?

Reply:
This repo only feeds a transport package to aggregator pipeline. Aggregator repo's pipeline includes only .dll and specific .exe files in producing the framework MSIX packages - this change is internal PR I am sending out shortly.